### PR TITLE
swimat 1.2.1 (new formula)

### PR DESCRIPTION
**Pull request by saagarjha**
Command line tool for Swimat, a Swift formatter
Note: Swimat is already available on Homebrew Cask as a Source Editor Extension (packaged as a .app). This formula is the command line tool only.

- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [N/A] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/swimat.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Swimat < Formula
+  desc "Command Line Tool to help format Swift code"
+  homepage "https://github.com/Jintin/Swimat"
+  url "https://github.com/Jintin/Swimat.git",
```
Comment (woodruffw) [inline Formula/swimat.rb]: Use the release URL instead: https://github.com/Jintin/Swimat/archive/v1.2.1.tar.gz

File: Formula/swimat.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    :tag => "v1.2.1"
+  sha256 "08876d0c13fc7782ada2cd2bbbf9724399ccea216caee0cfaf2acf298eb5b3ac"
+  head "https://github.com/Jintin/Swimat.git"
+
+  depends_on :xcode => "8.0"
+
+  def install
+    xcodebuild "-target", "CLI",
+    "-configuration", "Release",
```
Comment (woodruffw) [inline Formula/swimat.rb]: This and below lines of the `xcodebuild` should be aligned to the arguments of the first line.

**Review by woodruffw — DISMISSED** [review]

**Comment by saagarjha** [comment]
@woodruffw I've added the changes you requested.

**Comment by dunn** [comment]
We've been trying to avoid having software distributed both as a Cask and in homebrew-core.  What is the benefit of having just this CLI version instead of the .app?

**Comment by saagarjha** [comment]
@dunn The intended audience for the two is pretty disjoint. The app contains an Xcode Source Editor Extension, which is pretty unnecessary for those who *don't* use Xcode. This formula is just the command line tool–making it useless for those who *do* use Xcode.

**Comment by dunn** [comment]
OK, that makes sense to me.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by dunn** [comment]
Merged as 102de272eb6c6ec58a0a56bf355139687d083296, thanks!

**Pull request closed without merging**